sonar-db/src/main/java/org/sonar/db/user/UserDto.java

```diff
@@ -201,6 +202,21 @@ public UserDto setUpdatedAt(Long updatedAt) {
     return this;
   }
 
+  public boolean isRoot() {
+    return root;
+  }
+
+  /**
+   * Setters is not accessible as dedicated requests must be used to update the root flag of a user:
+   * <ul>
+   *   <li>a user can not be created root</li>
+   *   <li>the generic update method of a user can not change its root flag</li>
+   * </ul>
+   */
+  private void setRoot(boolean root) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "setRoot" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)